ui/input/src/plugins/voiceInput.ts

```diff
@@ -12,6 +12,16 @@ const speechLookup = new Map<string, string>([
   ['g', 'g'],
   ['h', 'h'],
 
+  // NATO phonetic alphabet
+  ['alpha', 'a'],
```

Comment:
Formally it’s noted `alfa` in the NATO alphabet ;) but maybe alpha get better recognized?

Reply:
Oh, good to know! I believe that the way the recognition works, it can only return words that are keys in this map. So I think that both would recognize equally well.